.github/workflows/test.yml

```diff
@@ -140,4 +140,15 @@ jobs:
     - run: docker ps  # show the services against which we'll be testing
     - run: npm install
     - run: npm ls --all || true
-    - run: npm test
+    - name: npm test
+      run: npm test
+    - name: debug test_output
+      run: ls test_output/*.* || true
+    - uses: v1v/otel-upload-test-artifact-action@v2
```

Comment:
This is my forked repo, waiting for the fix I put regarding JUnit to be merged https://github.com/inception-health/otel-upload-test-artifact-action/pull/9

So we can use it mine meanwhile

Reply:
```suggestion
    - uses: inception-health/otel-upload-test-artifact-action@v1
```